doc/userguide/rules/vlan-id.rst

```diff
@@ -0,0 +1,32 @@
+Vlan.id Keyword
+===============
+
+Suricata has a ``vlan.id`` keyword that can be used in signatures to identify
+and filter network packets based on Virtual Local Area Network IDs. By default,
+it matches all VLAN IDs. However, if a specific layer is defined, it will only match that layer.
+
+vlan.id uses :ref:`unsigned 32-bit integer <rules-integer-keywords>`.
```

Comment:
why 32 bit? Iirc we can have a 12bit number max, so should we use u16?

Reply:
ah I see the code uses u16

Reply:
indeed, doc should state u16 as used in the code

Reply:
Should we also mention here the max value, and the specification that defines that?